design-system/materials/internals/definition.yaml

```diff
@@ -101,6 +104,7 @@ choiceGroupsByTheme:
       label: Shadows
       prefix: shadow
       choices:
+        shadow-0: 'none'
```

Comment:
In the new theme we have buttons having no shadow at all so having a token for it makes sense to me.